airbyte-integrations/connectors/source-hubspot/metadata.yaml

```diff
@@ -67,4 +67,50 @@ data:
   tags:
     - language:python
     - cdk:python
+  connectorTestSuitesOptions:
+    - suite: unitTests
+    - suite: integrationTests
+      testSecrets:
+        - name: SECRET_SOURCE-HUBSPOT_CREDS
+          fileName: config.json
+          secretStore:
+            type: GSM
+            alias: airbyte-connector-testing-secret-store
+        - name: SECRET_SOURCE-HUBSPOT_OAUTH_CREDS
+          fileName: config_oauth.json
+          secretStore:
+            type: GSM
+            alias: airbyte-connector-testing-secret-store
+        - name: SECRET_SOURCE-HUBSPOT_OAUTH_NO_START_DATE_CREDS
```

Comment:
There are 2 cases for `config_oauth_no_start_date.json`, but we should probably include only 1 of them. I'm not sure which exactly. Maybe @maxi297 or @strosek could advise here?